test/test_helper.rb

```diff
@@ -136,6 +137,10 @@ def assert_deploy_success(result)
     alias_method :assert_restart_success, :assert_deploy_success
     alias_method :assert_task_run_success, :assert_deploy_success
 
+    def assert_render_success(result)
+      assert_equal true, result, "Render failed when it was expected to succeed"
```

Comment:
Let's also add an assertion that we've printed the correct result heading, like we have in the other `assert_[action]_success` helpers